app/controllers/api_controller/service_requests.rb

```diff
@@ -29,6 +29,17 @@ def service_requests_remove_resource(target, type, id, _data)
       end
     end
 
+    def find_service_requests(id)
+      klass = collection_class(:service_requests)
+      return klass.find(id) if @auth_user_obj.admin?
+      klass.find_by!(:requester => @auth_user_obj, :id => id)
+    end
+
+    def service_requests_search_conditions
+      return {} if @auth_user_obj.admin?
+      {:requester => @auth_user_obj}
+    end
+
```

Comment:
@imtayadeway how did this find service requests before and how does this filtering get invoked now?  I don't see either of these methods being called unless there's some metaprogramming magic.  Maybe you can explain what happened before and how this fixes it in the commit message? Or just educate me if I'm just missing something obvious.